app/allocation/views/view.njk

```diff
@@ -66,7 +78,32 @@
     {% endif %}
   </section>
 
-  <section class="govuk-!-margin-top-8">
+
+
+  {% if canAccess('allocation:person:assign') %}
+    <section class="govuk-!-margin-top-8">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">{{ t("allocations::view.people_assign.heading") }}</h2>
+    {% set slotsEmpty = allocation.moves_count - allocationPeople.filledSlots.length %}
+    {% if slotsEmpty %}
+    <p>You can’t add people who require a special vehicle to an allocation - you need to <a href="/move/new">create a single request</a> instead</p>
```

Comment:
Could use the GOV.UK warning text pattern here.

Reply:
We could - but that's not as designed.